### DISCUSSION TITLE
OSS

### DISCUSSION THREAD

**Discussion by originaxe** [teaxyz/white-paper#126] [General]
For context: We are a group of developers working in the private sector, but additionally in the last years we have launched a non-profit organization containing 4 projects, focused on different directions like audits and security, AI, data, web,... with tools and infrastructure solutions being widely used by many other projects and dependencies. At the moment we cumulatively have over 700 contributors (748 at the moment) that we would like to be rewarded through the tea protocol as well as participate in supporting and maintaining the tea protocol.

We have been researching the tea protocol for about a week, and we like the idea and value it can create for oss contributors. However, we have some questions that are not answered in the documentation and would like to get this information:

1. Protocol and governance:
- Will it be possible to register these 4 projects as one organisation under the tea protocol?
- Will there be a conflict of interest within the protocol or governance if as contributors to projects x, y, z, 0, we stake relatively large amounts in favour of these 4 projects and respectively within the governance we participate as one entity?

2. Team, core contributors and contributors:
- Will pkgx be registered as a project under the tea protocol?
- Are rewards foreseen for contributors (not core contributors) to pkgx/tea on mainnet (if pkgx is registered under the tea protocol) or at TGE?
- If pkgx will be registered under the tea protocol will rewards be distributed to core contributors?

3. The TEA token:
Around 19% (1.9 B) will be issued in circulation at TGE,
- Of which 50% to DAO, how will these tokens be used?
33% will be allocated towards public sale and liquidity, it is just a considered influencing factor.

And the last part of the TGE, 17% (323 M), will be allocated towards community, projects, devs as incentives and airdrop.
- How much will be allocated as incentives, and is it possible to know how much will be allocated to oss projects and how much to the community?
- How much will be allocated as airdrop, and which groups will be included in this airdrop?

Since incentives (rewards) are already foreseen for all ITN participants, the airdrop can be allocated to developers, which developers? Will these be the developers (not core) who contributed to pkgx/tea?
- How much will be allocated to community?

In order to know when and how to start to participate in the protocol (maintenance, security and governance), we need these details since the TEA token is the basic participation tool in protocol security and governance.

Sorry if we have asked too many questions and we hope to get answers.

**Comment by originaxe** [comment]
@thomas-borrel We hope you can help

**Reply by thomas-borrel** [reply]
Hi @originaxe,

Great questions! Comments are inline. 
We have been researching the tea protocol for about a week, and we like the idea and value it can create for oss contributors. However, we have some questions that are not answered in the documentation and would like to get this information:

1. Protocol and governance:
**Will it be possible to register these 4 projects as one organisation under the tea protocol?**
If you can find your projects using the Project teaRank page [here](https://tea.xyz/rewards-for-oss-contributions) and have the ability to commit + merge a file to each project’s repo, then yes. If your projects cannot be found in the Project teaRank page, let us know which package manager(s) you are using.

**Will there be a conflict of interest within the protocol or governance if as contributors to projects x, y, z, 0, we stake relatively large amounts in favour of these 4 projects and respectively within the governance we participate as one entity?**
Staking your own project is encouraged, and for project treasuries, a requirement. All project treasuries must stake a minimum of 25% of the teaRank rewards they receive to continue to receive teaRank rewards. This stake is used as a bounty pool to reward security researchers who report vulnerabilities to project contributors in accordance with the protocol’s requirements.

2. Team, core contributors and contributors:
**Will pkgx be registered as a project under the tea protocol?**
The pkgx package manager is a supported package manager at launch - i.e. all projects available through the pkgx package manager should have a teaRank, although not all will qualify for teaRank rewards. You can find out more about the tearank thresholds and limits here: https://docs.tea.xyz/tea/i-want-to.../learn-about-proof-of-contribution/what-is-proof-of-contribution-technical#rewards-thresholds-and-limits

**Are rewards foreseen for contributors (not core contributors) to pkgx/tea on mainnet (if pkgx is registered under the tea protocol) or at TGE?**
I’m not 100% clear on the question you are asking. Are you wondering if as a contributor to the pkgx package manager you would be eligible to some rewards?

**If pkgx will be registered under the tea protocol will rewards be distributed to core contributors?**
Same question as above.

3. The TEA token:
**Around 19% (1.9 B) will be issued in circulation at TGE, of which 50% to DAO, how will these tokens be used? 33% will be allocated towards public sale and liquidity, it is just a considered influencing factor.**
To clarify, ~50% of the tokens in circulation at TGE (i.e. ~50% of the 1.9B) are allocated to the DAO treasury and used to stake OSS projects and vote in protocol governance (Users wanting to participate to the governance of the tea Protocol must stake their TEA tokens to receive stTEA, used to calculate their voting power) as indicated [here](https://tea.xyz/tea-token).

**And the last part of the TGE, 17% (323 M), will be allocated towards community, projects, devs as incentives and airdrop. How much will be allocated as incentives, and is it possible to know how much will be allocated to oss projects and how much to the community?**

A very large portion of the maxSupply (10b tokens) is allocated to rewarding OSS projects. These tokens are distributed over several decades.
Are you looking for more clarity on this language from the [TEA token page](https://tea.xyz/tea-token) “The remaining 17% of TEA tokens to be distributed will be utilized to facilitate the protocol’s mainnet launch. This will include a strategic token sale to various institutional investors (11%), and a public sale to promote widespread community distribution at TGE (3%). The remaining ~3% of TEA tokens will be allocated to support marketplace liquidity once a token generation event occurs.”?
Also, note that we'll be publishing a more detailed tokenomics paper as we get closer to TGE

**How much will be allocated as airdrop, and which groups will be included in this airdrop?
Since incentives (rewards) are already foreseen for all ITN participants, the airdrop can be allocated to developers, which developers? Will these be the developers (not core) who contributed to pkgx/tea?**
These incentives are for all participants. Participants include all listed in the whitepaper including OSS developers, security researchers, project supporters/stakers, etc... 

**How much will be allocated to community?**
Per [this page](https://tea.xyz/tea-token), more than 50% of the TEA tokens created by the protocol are allocated to "Ecosystem & Governance" - If you chose to remove the allocation to the tea DAO (roughly 50% of the 1.9b you mentioned above or 9% of the maxSupply), it leaves with over 41% allocated to the community.

In order to know when and how to start to participate in the protocol (maintenance, security and governance), we need these details since the TEA token is the basic participation tool in protocol security and governance.
Sorry if we have asked too many questions and we hope to get answers.

No worries - I hope I provided clear answers.
